### PR TITLE
BONNIE-1248

### PR DESCRIPTION
**Pull request by c-monkey**
Back end `patient_builder_test` - replace QDM with CQL-based fixtures

Pull requests into Bonnie require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [X] This pull request describes why these changes were made.
- [X] This PR is into the correct branch.
- [X] JIRA ticket for this PR: https://jira.mitre.org/browse/BONNIE-1248
- [x] JIRA ticket links to this PR
- [X] Code diff has been done and been reviewed (it **does not** contain: additional white space, not applicable code changes, debug statements, etc.)
- [X] If UI changes have been made, google WAVE plug-in has been executed to ensure no 508 issues were introduced.
- [X] Tests are included and test edge cases
- [X] Tests have been run locally and pass (remember to update Gemfile when applicable)
- [X] Code coverage has not gone down and all code touched or added is covered. 
     * In rare situations, this may not be possible or applicable to a PR. In those situations:
         1. Note why this could not be done or is not applicable here: 
         2. Add TODOs in the code noting that it requires a test
         3. Add a JIRA task to add the test and link it here: 
- [X] Automated regression test(s) pass

If JIRA tests were used to supplement or replace automated tests:
- [X] JIRA test links: N/A
- [X] Justification for using JIRA tests: N/A
- [X] JIRA tests have been added to sprint  N/A

**Reviewer into feature branch**

Name: @losborne 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

If JIRA tests were used to supplement or replace automated tests:
- [X] JIRA tests have been run and pass  N/A
- [X] You agree with the justification for use of JIRA tests or have provided input on why you disagree  N/A

### CODE REVIEW

File: test/test_helper.rb [hunk 1]
```diff
@@ -3,6 +3,7 @@
 require File.expand_path('../../config/environment', __FILE__)
 require 'rails/test_help'
 require './lib/ext/record'
+require 'rake'
```
Comment (lukoktonos) [inline test/test_helper.rb:6]: Why was this line added?  The tests seem to pass for me without it.

Comment (c-monkey) [reply test/test_helper.rb:6]: For some reason it is required for me to run tests in the RubyMine debugger, as a known requirement. I don't mind removing it from this PR if you think it's problematic.

Comment (lukoktonos) [reply test/test_helper.rb:6]: Ah OK--tests work either way for me, so I don't mind.  Was just curious.

**Review by lukoktonos — CHANGES_REQUESTED** [review]
It looks like a couple of lines lost coverage with the updates:

image
image

EDIT: It was actually the other way around--coverage was _added_ for those two lines instead of removed.

**Review by lukoktonos — APPROVED** [review]
Looks good.

**Comment by c-monkey** [comment]
(I couldn't reproduce Luke's issue, and neither could he, so he approved the PR)

**Pull request merged**